Gruntfile.js

```diff
@@ -10,7 +10,7 @@ module.exports = function(grunt) {
                 options: {
                     mask: '*test.js',
                     coverageFolder: 'test/coverage',
-                    excludes: ['public', 'uploads', 'tmp', 'tpl']
+                    excludes: ['app', 'uploads', 'tmp']
                 }
             }
         },
```

Comment:
@ErisDS 
how can i verify/ensure that the deploy still works?
I can see that `touch tmp/restart.txt` triggers a restart, but how does passenger starts the app?
`npm start`?if so, the changes should not break the deploy.